deploy/olm-catalog/apicast-community-operator/apicast-community-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -5,15 +5,15 @@ metadata:
     capabilities: Full Lifecycle
     categories: "Integration & Delivery"
     certified: "false"
-    containerImage: quay.io/3scale/apicast-operator:master
+    containerImage: quay.io/3scale/apicast-operator:v0.1.0
     createdAt: "2019-10-27T22:40:00Z"
     description: APIcast is an API gateway built on top of NGINX. It is part of the Red Hat 3scale API Management Platform
     repository: https://github.com/3scale/apicast-operator
     support: Red Hat, Inc.
     alm-examples: >-
        [{"apiVersion":"apps.3scale.net/v1alpha1","kind":"APIcast","metadata":{"name":"example-apicast"},"spec":{"adminPortalCredentialsRef":{"name":"mysecretname"}}},
        {"apiVersion":"apps.3scale.net/v1alpha1","kind":"APIcast","metadata":{"name":"example-apicast"},"spec":{"embeddedConfigurationSecretRef":{"name":"mysecretname"}}}]
-  name: apicast-operator.v0.0.0
+  name: apicast-community-operator.v0.1.0
```

Comment:
Renamed the CSV name from apicast-operator to apicast-community-operator

Reply:
why not 
`v1.0.0` ???